### PR TITLE
install ARO with custom disk encryption set

### PR DESCRIPTION
**Pull request by mgahagan73**
Add a workflow where we create an azure disk encryption set, then install ARO configured to use the DES for control plane and worker nodes.

### CODE REVIEW

**Comment by mgahagan73** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aro-byok

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml [hunk 1]
```diff
@@ -61,6 +61,13 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-azure-ipi-private
+- as: installer-rehearse-aro-byok
```
Comment (jianlinliu) [inline ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml:64]:
```suggestion
- as: installer-rehearse-aro
```

**Comment by mgahagan73** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aro

**Comment by mgahagan73** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.14-installer-rehearse-aro

File: ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml (hunk 1 above)
Comment (mgahagan73) [reply ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.14.yaml]: job renamed.

**Comment by mgahagan73** [comment]
Confirmed DES was created and cluster using it successfully in previous rehearsal run. This should be ready to merge now.

**Pull request merged**